k8s/airflow-operator/README.md

```diff
@@ -0,0 +1,285 @@
+
+```shell
+kubectl apply -f "${APP_INSTANCE_NAME}_manifest.yaml" --namespace "${NAMESPACE}"
+```
+
+
+#### View the app in the Google Cloud Console
+
+To get the Console URL for your app, run the following command:
+
+```shell
+echo "https://console.cloud.google.com/kubernetes/application/${ZONE}/${CLUSTER}/${NAMESPACE}/${APP_INSTANCE_NAME}"
+```
+
+To view your app, open the URL in your browser.
+
+
+
+# Using Airflow Operator
+
+To deploy an Airflow cluster, we need to create the AirflowBase resource first followed by AirflowCluster resource.
+
+```
+# create AirflowBase resource first
+$ kubectl apply -f - <<EOF
+apiVersion: airflow.k8s.io/v1alpha1
+kind: AirflowBase
+metadata:
+  name: mc-base
+spec:
+  mysql:
+    operator: False
+EOF
+
+# get status of the CRs
+$ kubectl get airflowbase/mc-base -o yaml
+
+# after 30-60s deploy cluster components
+# create AirflowCluster resource next
+$ kubectl apply -f - <<EOF
+apiVersion: airflow.k8s.io/v1alpha1
+kind: AirflowCluster
+metadata:
+  name: mc-cluster
+spec:
+  executor: Celery
+  redis:
+    operator: False
+  scheduler:
+    version: "1.10.0rc2"
+  ui:
+    replicas: 1
+  worker:
+    replicas: 2
+  dags:
+    subdir: "airflow/example_dags/"
+    git:
+      repo: "https://github.com/apache/incubator-airflow/"
+      once: true
+  airflowbase:
+    name: mc-base
+EOF
+
+# get status of the cluster
+$ kubectl get airflowcluster/mc-cluster -o yaml
+
+# after 30-60s deploy port forward to access UI
+# access UI by pointing browser to localhost:8080
+$ kubectl port-forward mc-cluster-airflowui-0 8080:8080
+```
+
+# Uninstall the Application
+
+## Using the Google Cloud Platform Console
+
+1. In the GCP Console, open [Kubernetes Applications](https://console.cloud.google.com/kubernetes/application).
+
+1. From the list of applications, click **Airflow Operator**.
+
+1. On the Application Details page, click **Delete**.
+
+## Using the command line
+
+### Prepare the environment
+
+Set your installation name and Kubernetes namespace:
+
+```shell
+export APP_INSTANCE_NAME=airflow-operator-1
+export NAMESPACE=default
+```
+
+### Delete the resources
+
+> **NOTE:** We recommend to use a kubectl version that is the same as the version of your cluster. Using the same versions of kubectl and the cluster helps avoid unforeseen issues.
+
+To delete the resources, use the expanded manifest file used for the
+installation.
+
+Run `kubectl` on the expanded manifest file:
+
+```shell
+kubectl delete -f ${APP_INSTANCE_NAME}_manifest.yaml --namespace $NAMESPACE
+```
+
+Otherwise, delete the resources using types and a label:
+
+```shell
+kubectl delete application,statefulset,service \
+  --namespace $NAMESPACE \
+  --selector app.kubernetes.io/name=$APP_INSTANCE_NAME
+```
+
+### Delete the GKE cluster
+
+Optionally, if you don't need the deployed application or the GKE cluster,
+delete the cluster using this command:
+
+```
+gcloud container clusters delete "$CLUSTER" --zone "$ZONE"
+```
```

Comment:
Please remove empty line.

Reply:
i dont see an empty line (last line is just the backticks)